cypress/integration/features/deals/create-partnered-deal-wizard.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Create Partnered Deal
+  Background:
+    Given I navigate to create partnered deal wizard
+
+  Scenario: Go back to deal type select
+    Given I am presented with Partnered Deal proposal stage
+    When I go to previous step
+    Then I am presented the option to choose a partner
+
+  Scenario: I proceed from proposal stage without filling in required fields
```

Comment:
```suggestion
  Scenario: Proceed from proposal stage without filling in required fields
```

Maybe just preference, but Scenarios should be formulated in a general sense.
Exception: 
- Admin is not allowed to xyz
- others??

Reply:
Makes sense